deps/ada.h

```diff
@@ -1957,6 +1957,7 @@ template <typename E>
 #ifdef TL_EXPECTED_EXCEPTIONS_ENABLED
   throw std::forward<E>(e);
 #else
+  (void)e;
```

Comment:
Can you open a PR to upstream this change to Ada's repository?

Reply:
Oh, I didn't realise those are copies. Sure.

Reply:
See conversation below.